GOVERNANCE.md

```diff
@@ -0,0 +1,192 @@
+perform code reviews and work towards becoming a *core-contributor* for the
+subproject/repo that they are active in.
+
+### Core-Contributor
+
+Core-Contributors are able to both review and approve code contributions as well as 
+help maintainers triage issues and with project management.
+
+While code review is focused on code quality and correctness, approval is focused on
+holistic acceptance of a contribution including: backwards / forwards
+compatibility, adhering to API and flag conventions, subtle performance and
+correctness issues, interactions with other parts of the system, etc.
+
+**Defined by:** *approvers* entry in an OWNERS file in a repo owned by the
+Atlantis project.
+
+#### Requirements
+
+The following applies to the part of the codebase for which one would be an approver in an `OWNERS` file.
+
+- Member for at least 3 months
+- Author of at least 3 substantial PRs to the codebase, with the
+  definition of substantial subject to the lead's discretion (e.g.
+  refactors, enhancements rather than grammar correction or one-line pulls).
+- Exhibiting sound technical judgment through PR contributions
+- Exhibiting sound technical judgment through PR reviews
+- Sponsored by a Maintainer
+  - With no objections from other Maintainers or Steering Committee
+  - Done through PR to update the OWNERS file
+
+
+#### Responsibilities and privileges
+
+The following applies to the part of the codebase for which one would be a core-contributor
+in an `OWNERS` file.
+
+- Core-contributor status may be a precondition to accepting large code contributions
+- Demonstrate sound technical judgment
+- Responsible for project quality control via code reviews
+  - Focus on holistic acceptance of contributions such as dependencies with other features, backward / forwards
+    compatibility, API and flag definitions, etc
+- Expected to be responsive to review requests (inactivity may result in suspension until active again)
+- Mentor Members
+- May approve and merge code contributions for acceptance
+- Monitor Atlantis Slack (delayed response is perfectly acceptable), particularly for the area of your repository
+- Periodically attend the recurring community meetings
+- In general, continue to be willing to spend at least 5% of their time working on Atlantis (~0.25 business days per week)
+
+Each repository's current list of core-contributors is published and updated in each repo’s OWNERS.md file.
+
+### Maintainer
+
+Each repository in the Atlantis organization are allowed their own unique set of maintainers. Maintainers have the most experience with the given repo and are expected to have the knowledge and insight to lead its growth and improvement.
+
+New maintainers and subproject maintainers must be nominated by an existing maintainer and must be elected by a supermajority of existing maintainers. Likewise, maintainers can be removed by a supermajority of the existing maintainers or can resign by notifying one of the maintainers.
```

Comment:
`one of the maintainers` is a bit fuzzy in expectation compared to many other statements in the document; should folks resigning perhaps notify the existing maintainer group _and_ the steering committee, so its tracked and verified the maintainer stepping down is properly having all permissions and access revoked - and not accidentally being forgotten by "one of the maintainers" - that potentially also could be churning out of the project